### PR TITLE
Added Github Profile Card Snippet

### PR DESCRIPTION
**Pull request by 1hanzla100**

### CODE REVIEW

**Comment by 1hanzla100** [comment]
@saadpasta Kindly Review PR.

File: Github Profile Card/Readme.md [hunk 1]
```diff
@@ -0,0 +1,16 @@
+# Github Profile Card
+
+## Preview
+<img src="Picture.PNG"></img>
```
Comment (saadpasta) [inline Github Profile Card/Readme.md:4]: Image not available in preview

**Comment by 1hanzla100** [comment]
Picture
@saadpasta

**Comment by 1hanzla100** [comment]
@saadpasta It is appearing in preview
Capture

**Review by saadpasta — APPROVED** [review]

**Comment by saadpasta** [comment]
@allcontributors please add @1hanzla100 for code

**Pull request merged**